### DISCUSSION TITLE
TypeError: expected str, bytes or os.PathLike object, not NoneType，How to solute this problem..

### DISCUSSION THREAD

**Discussion by EEEEEREN** [STVIR/pysot#455] [Q&A] [answered]
我是在windows10上，跑的此代码，并没有训练数据集以及测试，我只是想测试一个小demo
运行之后出现如此报错
![QQ截图20201020144740](https://user-images.githubusercontent.com/67900018/96550337-4f2a9900-12e3-11eb-8287-5a2b855a79f1.png)
是不是此代码在linux下运行更好

**Comment by ZhiyuanChen** [comment, marked as answer]
> 运行之后出现如此报错

你没有指定一个配置文件

> 是不是此代码在linux下运行更好

是

**Reply by EEEEEREN** [reply]
请问这个配置文件在哪呢。。 原代码里面也没有呀

**Reply by ZhiyuanChen** [reply]
请先阅读 https://github.com/STVIR/pysot/blob/master/README.md
